metrics/collector.go

```diff
@@ -58,6 +66,10 @@ func (c *DefaultCollector) ApiErrorOccurred() {
 	c.apiErrorsCounter.Inc()
 }
 
+func (c *DefaultCollector) EvmBlockIndexed() {
+	c.evmBlockHeight.Inc()
```

Comment:
we should take in the height as a value, there might be cases where we will re-index already indexed values (crash recovery)